src/models/lesson.js

```diff
@@ -53,6 +53,20 @@ const lessonSchema = new Schema(
         message: ENUM_CAN_BE_ONE_OF('resource type', RESOURCES_TYPES_ENUM)
       },
       default: RESOURCES_TYPES_ENUM[0]
+    },
+    availability: {
+      status: {
+        type: String,
+        enum: {
+          values: RESOURCE_AVAILABILITY_STATUS_ENUM,
+          message: ENUM_CAN_BE_ONE_OF('resource availability status', RESOURCE_AVAILABILITY_STATUS_ENUM)
+        },
+        default: RESOURCE_AVAILABILITY_STATUS_ENUM[0]
+      },
+      date: {
+        type: Date,
+        default: null
+      }
```

Comment:
Did you think about reusing it? Because it is repeated in `quizzes`, `lessons`, `attachments`

Reply:
Yes, I thought about it. But I don't have any idea where should I take this file. We don't have any files for constants of this type.  If you have an idea, please tell me